plugin/src/main/groovy/redis/RedisGrailsPlugin.groovy

```diff
@@ -10,7 +10,7 @@ import redis.clients.jedis.Protocol
 
 class RedisGrailsPlugin extends Plugin {
 
-    def grailsVersion = "3.0.0 > *"
+    def grailsVersion = "7.0.0 > *"
```

Comment:
set this to `7.0.0-SNAPSHOT` for now

Reply:
I just recently noticed the log message that appears when testing with Grails 7.0.0-SNAPSHOT